### DISCUSSION TITLE
https://github.com/Hoot-Coding revival anybody?

### DISCUSSION THREAD

**Discussion by rubiidev18alt** [cytrinc/bark-coding#263] [General]
eh? eh? (start off with a bark fork, and change stuff to be based off hoot)

**Comment by raynepaws** [comment]
"revival" implies it was once alive
![image](https://github.com/user-attachments/assets/d274307a-c845-469e-b8c6-3f6f9f2bd502)
> (start off with a bark fork, and change stuff to be based off hoot)

like microsoft edge? take google chrome and reskin everything? if you want to, go ahead, but i've got work to do here

i'm sorry but i fail to see how this is related to bark

**Comment by mariocraft987** [comment]
the name "hoot" is meh, too much scope
